gensim/models/doc2vec.py

```diff
@@ -580,7 +580,7 @@ def __init__(self, documents=None, dm_mean=None,
         need about 1GB of RAM. Set to `None` for no limit (default).
 
         `sample` = threshold for configuring which higher-frequency words are randomly downsampled;
-                default is 1e-3, useful value is 1e-5.
+                default is 1e-3, values to 1e-5 (or lower) may also be useful.
```

Comment:
About "no downsampling" case, see examples

with `sample = 0.999999`
```
INFO:gensim.models.word2vec:sample=0.999999 downsamples 0 most-common words
```

with `sample = 1.`
```
INFO:gensim.models.word2vec:sample=1 downsamples 21143 most-common words
```

The reason is [alternative logic](https://github.com/menshikh-iv/gensim/blob/30a8c8f2fbbc95585b2322992bdd8f09ba29fa12/gensim/models/word2vec.py#L672) when `sample >= 1`